### PR TITLE
Sortable Education && publish and finish ad optional

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

File: addons/default/visiosoft/advs-module/resources/views/new-ad/new-create.twig [hunk 1]
```diff
@@ -134,6 +137,96 @@
...
+                                                font-size: 1.25rem;
+                                                border-radius: 0.25rem;
+                                            }
+
+                                            .input-group-addon input[type=radio],
+                                            .input-group-addon input[type=checkbox] {
+                                                margin-top: 0;
+                                            }
+
+                                        </style>
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/views/new-ad/new-create.twig:226]: why inline style?

File: addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php [hunk 2]
```diff
@@ -272,7 +275,18 @@ public function getEducation(Request $request)
...
+                    'profile_education_part_translations',
+                    'profile_education_part.id',
+                    '=',
+                    'profile_education_part_translations.entry_id'
+                )
+                ->where('profile_education_part_translations.locale', '=', Request()->session()->get('_locale', setting_value('streams::default_locale')))
+                ->where('education_id', $request->education)
+                ->selectRaw('default_profile_education_part.*, default_profile_education_part_translations.name as name')
+                ->orderBy('name', 'ASC')
+                ->get();
```
Comment (diashalabi) [inline addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php:289]: why left join?

**Review by diashalabi — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php (hunk 2 above)
Comment (spektra2147) [reply addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php:289]: for order by name

**Review by muammertop — APPROVED** [review]

File: addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php (hunk 2 above)
Comment (diashalabi) [reply addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php:289]: can't you just do EducationPartModel::query()->where('education_id', $request->education)->get()->sortBy('name'); ?

**Review by diashalabi — CHANGES_REQUESTED** [review]

**Comment by spektra2147** [comment]
Thank you for the sortBy function @diashalabi

**Review by diashalabi — APPROVED** [review]

**Comment by diashalabi** [comment]
> Thank you for the sortBy function @diashalabi

no problem 😉

**Pull request merged**